applications/nrf_desktop/board_configuration.rst

```diff
@@ -102,7 +102,7 @@ Sample dongle (``nrf5340dk/nrf5340``)
         Input data comes from Bluetooth and is retransmitted to USB.
       * The configuration with the B0 bootloader is set as default.
 
-Sample mouse or keyboard (``nrf54l15pdk/nrf54l15/cpuapp``, ``nrf54l15pdk@0.3.0/nrf54l15/cpuapp``)
+Sample mouse or keyboard (``nrf54l15pdk/nrf54l15/cpuapp``)
```

Comment:
Should we also mention `nrf54l15pdk/nrf54l15/cpuapp@0.2.1` (it's still supported)?

Reply:
In sample.yaml we do not mention it, so other samples/apps do not mention it either if they use it or not.

Reply:
Ok, then leave as is ("silent support")